haystack/components/rankers/transformers_similarity.py

```diff
@@ -106,28 +121,57 @@ def to_dict(self) -> Dict[str, Any]:
             top_k=self.top_k,
             meta_fields_to_embed=self.meta_fields_to_embed,
             embedding_separator=self.embedding_separator,
+            scale_score=self.scale_score,
+            calibration_factor=self.calibration_factor,
+            score_threshold=self.score_threshold,
             model_kwargs=self.model_kwargs,
         )
 
     @component.output_types(documents=List[Document])
-    def run(self, query: str, documents: List[Document], top_k: Optional[int] = None):
+    def run(
+        self,
+        query: str,
+        documents: List[Document],
+        top_k: Optional[int] = None,
+        scale_score: Optional[bool] = None,
+        calibration_factor: Optional[float] = None,
+        score_threshold: Optional[float] = None,
+    ):
         """
         Returns a list of Documents ranked by their similarity to the given query.
 
         :param query: Query string.
         :param documents: List of Documents.
         :param top_k: The maximum number of Documents you want the Ranker to return.
+        :param scale_score: Whether the raw logit predictions will be scaled using a Sigmoid activation function.
+            Set this to False if you do not want any scaling of the raw logit predictions.
+        :param calibration_factor: Factor used for calibrating probabilities calculated by
+            `sigmoid(logits * calibration_factor)`. This is only used if `scale_score` is set to True.
+        :param score_threshold: If provided only returns documents with a score above this threshold.
         :return: List of Documents sorted by their similarity to the query with the most similar Documents appearing first.
         """
         if not documents:
             return {"documents": []}
 
         if top_k is None:
             top_k = self.top_k
-
         elif top_k <= 0:
             raise ValueError(f"top_k must be > 0, but got {top_k}")
 
+        if scale_score is None:
+            scale_score = self.scale_score
+
+        if calibration_factor is None:
+            calibration_factor = self.calibration_factor
+
+        if scale_score and calibration_factor is None:
+            raise ValueError(
+                f"scale_score is True so calibration_factor must be provided, bug got {calibration_factor}"
+            )
```

Comment:
Lets use those or one liners, more compact

Reply:
Sorry how do you mean exactly, I'm a little unsure about the requested change

Reply:
scale_score = scale_score or self.scale_score